fv3core/requirements/requirements_base.txt

```diff
@@ -1,5 +1,5 @@
 f90nml
-pytest-xdist
+pytest-xdist<2  # last version that works with pytest 5
```

Comment:
pytest 5 is required by gt4py. It looks like something changed and now gt4py requirements are picked up when compiling constraints.txt.